content/en/checkout/demo.md

```diff
@@ -5,4 +5,6 @@ position: 31
 category: Checkout
 ---
 
-Conteúdo
\ No newline at end of file
+You can explore a [live integration of Checkout](https://checkout-demo.easypay.pt) to see for yourself what exactly is included in the solution, as well as trying out each payment flow.
+
```

Comment:
`You can explore a [live integration of Checkout](https://checkout-demo.easypay.pt) to see` **(remove: for yourself)** `what exactly is included in the solution, as well as trying out each payment flow.`